openfe/setup/methods/base.py

```diff
@@ -0,0 +1,34 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import abc
+from openff.toolkit.utils.serialization import Serializable
+
+
+class FEMethod(Serializable):
```

Comment:
We were getting an MRO on this, Serializable already inherits from ABC so passing both here shouldn't be necessary?

Reply:
It's because some of the methods (e.g. `to_toml` iirc) of `Serializable` aren't defined so you can't go deep on the inheritance.  I think you need to include `abc.ABC` again to make it clear that this is still technically an abstract class